src/include/connection.h

```diff
@@ -214,6 +214,8 @@ struct __wt_connection_impl {
     WT_LSN *debug_ckpt;      /* Debug mode checkpoint LSNs. */
     uint32_t debug_ckpt_cnt; /* Checkpoint retention number */
 
+    bool debug_realloc_exact; /* Reallocate exact size */
+
```

Comment:
Similar to the other one, no need to expand the connection structure for this. You can add a bit in the flags `WT_CONN_DEBUG_REALLOC_EXACT`.

Reply:
I'll add this after WT-4921 is merged in case the automatic flag value generation clashes